CONTRIBUTING.rst

```diff
@@ -18,12 +18,71 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, PEPs that are not marked as Draft or Active are usually considered
```

Comment:
Remove one of "In general" and "usually". Come to think of it, probably remove both.

Reply:
I removed the latter; I wanted to hedge a bit here due to provisional PEPs, some PEPs that are de-facto specifications and updated from time to time, and other edge cases—though there has been a _general_ push to reduce that over time and move most of that elsewhere to more suitable platforms.